### PR TITLE
Update phables to 1.4.1

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`phables`](https://bioconda.github.io/recipes/phables/README.html): **1.4.0** &rarr; **1.4.1**

[install with bioconda](http://bioconda.github.io/recipes/phables/README.html) [Conda](https://anaconda.org/bioconda/phables/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/phables`](https://github.com//bioconda/bioconda-recipes/tree/bump/phables/recipes/phables) (click to view/edit other files)
Summary | Phables: from fragmented assemblies to high-quality bacteriophage genomes
Home | [https://github.com/Vini2/phables](https://github.com/Vini2/phables)
Releases |[https://github.com/Vini2/phables/tags](https://github.com/Vini2/phables/tags)
Recipe Maintainer(s) | @Vini2
Author | @Vini2
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by Vini2** [comment]
Please don't merge this PR yet. I have to edit it to skip the Windows build. 

Thanks!

**Comment by mbhall88** [comment]
@Vini2 I have converted it to a draft so no one accidentally merges it. Convert it back when you are happy with your edits and I will approve.

**Comment by Vini2** [comment]
Thanks @mbhall88! I've updated the recipe and now it's ready for review.

I followed the edits done in PR [`50961`](https://github.com/bioconda/bioconda-recipes/pull/50961) to skip the Windows build.

**Comment by Vini2** [comment]
@BiocondaBot please add label

File: recipes/phables/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
 source:
   url: "https://github.com/Vini2/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz"
-  sha256: 09b843141ab32cda5016741b7f31ccefcb659448f0fd4aa55697c509680e16e9
+  sha256: f456260bf710073986ecd524975e09083c5cf7d0ad3a0e6808ba48a2b7dc0ee4
 
 build:
   number: 0
-  noarch: python
+  noarch: false
```
Comment (martin-g) [inline recipes/phables/meta.yaml:14]: What is the reason for this change ?
I see no arch specific binaries built by the recipe: https://github.com/bioconda/bioconda-recipes/pull/51124#issuecomment-2390765305

Comment (Vini2) [reply recipes/phables/meta.yaml:14]: I only wanted to skip the Windows build as an [environment](https://github.com/Vini2/phables/blob/develop/phables/workflow/envs/smg.yaml) created in phables uses [fraggenescan](https://anaconda.org/bioconda/fraggenescan) and [hmmer](https://anaconda.org/bioconda/hmmer) which only support linux and osx.

Comment (mbhall88) [reply recipes/phables/meta.yaml:14]: Normally you would use the [`build/skip` entry](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#skipping-builds) if you want to skip windows.

Comment (mbhall88) [reply recipes/phables/meta.yaml:14]: Though it does also say that if you shouldn't use a preselector (e.g., `# [win]`) with `noarch` packages

Comment (martin-g) [reply recipes/phables/meta.yaml:14]: Bioconda does not build for Windows anyway. Only Linux and OSX are supported at the moment.

Comment (Vini2) [reply recipes/phables/meta.yaml:14]: So what would be the best way to specify this in the recipe?

Comment (martin-g) [reply recipes/phables/meta.yaml:14]: Honestly I do not see a better solution than the current one.
You want to avoid Windows users to try to install the package, so you dropped `noarch`.

If you decide to go back to `noarch` then you can tell the users that Bioconda does not support Windows. It is written at the very top of the home page (https://bioconda.github.io/)
